src/main.c

```diff
@@ -1766,6 +1766,16 @@ int validate_and_call(yubihsm_context *ctx, CommandList l, const char *line) {
   return 0;
 }
 
+static void free_configured_connectors(yubihsm_context *ctx) {
+  if (ctx->connector_list) {
```

Comment:
I would change to `if (ctx && ctx->connector_list)`

Reply:
Yeah, except other similar functions don't do it...